gateway/ln-gateway/src/lib.rs

```diff
@@ -938,6 +943,7 @@ impl Gateway {
                 .write()
                 .await
                 .insert(mint_channel_id, federation_id);
+            info!("SCID {mint_channel_id} assigned to {federation_id}");
```

Comment:
Let's make this a little more informative. Can we add some statement that says a new federation was connected? And perhaps we want to move this to below `save_config`. It might be kind of confusing to see this message and then have `save_config` fail for whatever reason

Reply:
Updated. I've also dropped log level to debug because this handler returns an informative result immediately after the log point